app/helpers/priceFormatter.js

```diff
@@ -0,0 +1,9 @@
+export const formatPrice = (priceValue) => {
+  const formattedPrice = priceValue.toLocaleString(undefined, {
+    minimumFractionDigits: 3, maximumFractionDigits: 3,
+  });
+
+  const truncatedTo2dp = formattedPrice.substring(0, formattedPrice.indexOf('.') + 3);
```

Comment:
do we need to keep waht the original value is? or else we're losing data

Reply:
mmm good shout but I don't think so.. This data will come from ordapi which will have the original value and for the preview we never update the order so it will just be for display purposes only